es-core/src/components/ImageGridComponent.h

```diff
@@ -40,7 +40,7 @@ class ImageGridComponent : public IList<ImageGridData, T>
 	using IList<ImageGridData, T>::getTransform;
 	using IList<ImageGridData, T>::mSize;
 	using IList<ImageGridData, T>::mCursor;
-	using IList<ImageGridData, T>::Entry;
+	using IList<ImageGridData, T>::mEntry;
```

Comment:
Changed to mEntry and updated class IList to include a protected member called mEntry of type struct Entry.